### ISSUE TITLE
Choose a appropriate UI framework

### ISSUE THREAD

**Issue by yinzixie** [Nymphxyz/wenc-frontend#11]
I am struggling with the choice of the UI framework.

**Comment by yinzixie** [comment]
In the beginning, I chose Ant Design Vue, but I can't figure out how to integrate it with `vitest`. And their documentation is not good. And no auto-import `css` with `vite`.

Then I tried Element Plus, they have auto-import and better doc in Chinese but it's ridiculous I can't assign style and class on their component. Feels really inconvenient, and lose some flexibility.

I can see Vuestic is more popular but v2 is not supported vue3 and they are still not releasing their v3.0 version and look like they won't release it in 2022. In addition, v3 only has a few components supported. 
https://github.com/vuetifyjs/vuetify/discussions/14776
https://github.com/vuetifyjs/vuetify/issues/14729

After some further investigation now I would prefer to use Quasar. It has good documentations for both usage and testing. Crossed plaformed and not only target to desktop.

I will give it a try and see what to do next step.
